Overlays/Posix/Libraries/WNFileSystem/CMakeLists.txt

```diff
@@ -42,12 +42,6 @@ wn_check_include_file("fcntl.h" WN_HAS_FCNTL_H)
 
 set(WN_HAS_FTRUNCATE_ALTERNATIVE false)
 
-wn_check_symbol_exists(open "fcntl.h" WN_HAS_OPEN_SYMBOL)
```

Comment:
On android the `open` symbol becomes ambiguous with how cmake checks for symbol existence and the check fails. I'll eventually just write a custom test for this on android. Removing for now.